apps/webbsite/src/pages/blog.tsx

```diff
@@ -0,0 +1,53 @@
+import { GetStaticProps } from 'next';
+import { FC } from 'react';
+import { BlogSection, FeaturedPostSection, Heading2 } from '../components';
+import { Notion, Post, Video } from '../libs/notion';
+
+type BlogProps = {
+  posts: Post[];
+  videos: Video[];
+};
+
+const Blog: FC<{ blog: BlogProps }> = ({ blog }) => {
+  const { posts, videos } = blog;
+
+  // Featured post is the last one in the array
+  const featuredPost = posts[posts.length - 1];
+  // Recent videos are the last 3 in the array
+  const recentVideos = videos.slice(-3);
```

Comment:
Consider adding checks to ensure that featuredPost and recentVideos have valid values:

```
  const featuredPost = posts.length > 0 ? posts[posts.length - 1] : null;
  const recentVideos = videos.length > 0 ? videos.slice(-3) : [];
```